packages/sdk/src/contracts/prebuilt-implementations/marketplace.ts

```diff
@@ -438,6 +438,29 @@ export class Marketplace implements UpdateableNetwork {
     await this.contractWrapper.multiCall(encoded);
   }
 
+  /**
+   * returns the minimum bid a user can place to outbid the previous highest bid
+   * @param listingId - the listing id of the auction
+   */
+  public async getMinimumNextBid(
+    listingId: BigNumberish,
+  ): Promise<BigNumberish> {
+    const currentAuction = this.auction;
+    const currentBidBuffer = await this.getBidBufferBps();
+    const winningBid = await currentAuction.getWinningBid(listingId);
+    const activeListing: AuctionListing = await currentAuction.getListing(
+      listingId,
+    );
+    if (currentAuction && winningBid) {
```

Comment:
currentAuction should always be defined right? also if you move it to `MarketplaceAuction` you won't need this